packages/core/svelte/use-chat.ts

```diff
@@ -60,7 +60,7 @@ export function useChat({
   body
 }: UseChatOptions = {}): UseChatHelpers {
   // Generate a unique ID for the chat if not provided.
-  const chatId = id || `chat-${uniqueId++}`
+  const chatId = id || `chat-${uniqueId}`
```

Comment:
this isn't calling the function, it's just reusing the same one.

Reply:
Great catch! I'll fix it now.